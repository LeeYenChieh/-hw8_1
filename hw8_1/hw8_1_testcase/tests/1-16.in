YKVb1PX6TnhBl9EswymhJuvo2zsynUoBiEbPDgPxhvy4mKMN5Z4IwZgzffdyy xXaDyOkGjRgFYefOFCVvybr4M8M56Z8Q3wkbmv1xigpCUDHxb
t8HXA7cFmLqVwfWhVQ03zylqSaK1YE bnc2J6oq2cA2gW2L3Axik
9a54k8fCxymkWZnu5IZOK CCccnfr40AjY5W4GKqd8TTd27i1uMQ3j
bgxcGfVaAp2w ZuT2HCmlDW4ROiCexgXd41Q0LojcwyILfnQwxDM
J8YqV7pSfN7LRi6NLJIMrZBYUw6aKANHCF4yE61mgdHBABRwGgj6 pVIrDOXYOezXL
Rpmsf6jdYIieQxzzxHDR0e6 1Drr2yl34
eUitIqGmIXTNpJ1a03QHT642BckkNWjOkFeEPl6ajJN42ZujcUHdhOZBEutZV T0vv9OIQsAV42DbSVpYfe9X8X55pa4YdoXOEnKi
9a54k8fCxymkWZnu5IZOK WMe89yVRAllJtFofIOOKqsMrWl9eSoyxJV7G0i9
9a54k8fCxymkWZnu5IZOK display
Rpmsf6jdYIieQxzzxHDR0e6 display
JZpp1ckz5rN4 7RDSvGcn9Hm7g
ktSQuEzr4xeA8P3LJD1OWSBhkWjmi J7rdvpwRhh4vcgrJG
YKVb1PX6TnhBl9EswymhJuvo2zsynUoBiEbPDgPxhvy4mKMN5Z4IwZgzffdyy display
bgxcGfVaAp2w add scrollBar thickBlackBorder
ng0RUKuKeTmQYtxBoVMDL 9ohQpTi8LMawgxFCc7QgjV5JUl4Ufd04ooYzPUMX
