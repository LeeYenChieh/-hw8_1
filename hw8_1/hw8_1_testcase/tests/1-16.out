WMe89yVRAllJtFofIOOKqsMrWl9eSoyxJV7G0i9 
1Drr2yl34 
xXaDyOkGjRgFYefOFCVvybr4M8M56Z8Q3wkbmv1xigpCUDHxb 
